app/src/store/playground/playgroundStore.tsx

```diff
@@ -83,22 +83,17 @@ export const _resetToolId = () => {
 const generateChatCompletionTemplate = (): PlaygroundChatTemplate => ({
   __type: "chat",
   messages: [
-    {
```

Comment:
This is the default system prompt for some other frameworks. I tend to find having the slot available to edit to be a nice default...